.travis.yml

```diff
@@ -54,3 +54,4 @@ script:
   - echo "JavaScript tests start here"
   - docker-compose -f docker/docker-compose.test.yml -p listenbrainz_test run --rm frontend_tester npm test
   - docker-compose -f docker/docker-compose.test.yml -p listenbrainz_test run --rm frontend_tester npm run type-check
+  - docker-compose -f docker/docker-compose.test.yml -p listenbrainz_test run --rm frontend_tester npm run format
```

Comment:
Does this actually format the file?

The behaviour in travis I'd expect is for this return a non-zero exit code if there's any file which is not linted correctly and fail the build.

Reply:
It doesn't actually format the files, a non-zero exit code along with the errors made will be returned.

Reply:
Sgtm, thanks for clarifying!